core/server/apps/subscribers/lib/views/subscribe.hbs

```diff
@@ -25,7 +25,7 @@
             <div class="gh-flow">
                 <header class="gh-flow-head">
                     <nav class="gh-flow-nav">
-                        <a href="{{@blog.url}}" class="gh-flow-back"><i class="icon-arrow-left"></i> Back</a>
+                        <a href="{{@blog.url}}" class="gh-flow-back" onclick="window.history.go(-1); return false;"><i class="icon-arrow-left"></i> Back</a>
```

Comment:
yeah this what what i thought, but when visiting:
`http://localhost:2368/2016/06/07/welcome-to-ghost/` pressing menu->subscribe `subscribed_url` is empty